lua/entities/gmod_tardis/modules/sh_time_distortions.lua

```diff
@@ -1,12 +1,21 @@
 -- Support for time distortion generator by parar020100 and JEREDEK
 
-local search_radius = 1000
+local search_radius = 2048
 local explode_radius = 300
 
 local function TimeDistortionsPresent(pos, radius)
     for i,v in ipairs(ents.FindInSphere(pos, radius)) do
-        if v:GetClass() == "gmod_time_distortion_generator" and v:GetEnabled() then
-            return true
+        if(v:GetClass() == "gmod_time_distortion_generator" and v.On) then
+            if(radius == search_radius) then
+                print(v:GetPos():Distance(pos))
+                if(v:GetPos():Distance(pos) <= v.Radius) then
+                    return true
+                end
+            else
+                if(v:GetPos():Distance(pos) <= radius) then
```

Comment:
General comment about the if statements across all the changes they seem to have no space after the `if` and they have unnecessary brackets

Reply:
I personally find it easier to read this way, and it just feels better (probably because I started coding as an E2 user) but I can change it if you'd like

Reply:
If you could please - thanks, then it matches Lua conventions and the conventions of the TARDIS codebase